kubeflow/gcp/setup_backend.sh

```diff
@@ -22,9 +22,19 @@ gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS
 gcloud config list
 
 NODE_PORT=$(kubectl --namespace=${NAMESPACE} get svc ${SERVICE} -o jsonpath='{.spec.ports[0].nodePort}')
+echo "node port is ${NODE_PORT}"
+
+while [[ -z ${BACKEND_NAME} ]]; do
+  BACKENDS=$(kubectl --namespace=${NAMESPACE} get ingress ${INGRESS_SERVICE} -o jsonpath='{.metadata.annotations.ingress\.kubernetes\.io/backends}')
```

Comment:
check the new env var not empty at line 6.

Reply:
done.